### DISCUSSION TITLE
Can I use Standard Notes in China?

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2712] [General]
hi i'm planning go to China and i want to use Standard Notes but is it safe to use Standard Notes in China because i will using Chinese network and is Standard Notes even works in Mainland China?

**Comment by DCI007** [comment]
We just returned to the States from China this year and, as of June, Standard Notes works in China. You won't have any problems with syncing or anything like that. I don't know what you are asking about when you say "is it safe to use in China", so I'll answer this a couple of different ways: First, SN is not illegal to my knowledge, so you won't face any problems from the CCP just for having it on your devices; Second, the encryption is sound enough to keep your data safe from the average intrusion efforts; Third, if your device is confiscated, however, and the CCP wants the data you are storing in Standard Notes, then you will be forced to log into your device and give up the data. Even if you somehow managed to refuse, if the CCP wanted to break SN's encryption, they probably could. 

Bottom line is that the answer to your question depends on what you mean by "safe". I used the free version of SN in China and it met my needs and kept my data safe from general intrusion. But, if you are worried about dealing with the CCP, the only way to keep your data safe from them is to delete it before they get to it. 

Also, bear in mind, while visiting China it is best to compartmentalize your apps on two different phones. Carry one phone with your foreign apps (like Facebook, VPN, Standard Notes, etc), and then carry one local phone with your Chinese apps (like WeChat, AliPay, DiDi, etc). WeChat is basically advanced Spyware and I wouldn't be surprised if it was capable of breaking SN security.

**Comment by JaspalSuri** [comment]
> Third, if your device is confiscated, however, and the CCP wants the data you are storing in Standard Notes, then you will be forced to log into your device and give up the data. Even if you somehow managed to refuse, if the CCP wanted to break SN's encryption, they probably could.

It would actually be easier to them to install some sort of spyware on a user's device when it's confiscated. Once the device is compromised, then any level or form of encryption is rendered moot for any app or service. They would then hand it back over and wait if needed. 

Thank you so much for your input, @DCI007!

**Comment by ghost** [comment]
Thank you Guys for the input i will consider those advice hope i don't get flagged by the authorities

**Comment by chaserene** [comment]
> if the CCP wanted to break SN's encryption, they probably could.

@DCI007 it's unlikely. if the CCP could brute-force XChaCha20-Poly1305, we would have more serious things to worry about. what's more probable is what you described, they would [force the victim](https://xkcd.com/538/) to reveal their password, or exploit some stupid vulnerability in the operating system to retrieve keys or decrypted data.
